app/views/faq_groups/platform.md

```diff
@@ -15,7 +15,7 @@ However, only the Core Team can award a bounty.
 
 # Where can I request features for Assembly?
 
-You can [submit a bounty](/meta/wips), or you can [hop into general chat](www.assembly.com/chat/general) and let us know.
+You can [submit a bounty](/meta/wips), or you can [hop into general chat](/chat/general) and let us know.
```

Comment:
1. Should head to `/meta/bounties`, just so we’re not relying on an old redirect
2. I think it’s called “community chat” now :)